allauth/account/tests.py

```diff
@@ -301,27 +330,6 @@ def test_ajax_login_success(self):
         data = json.loads(resp.content.decode('utf8'))
         self.assertEqual(data['location'], '/accounts/profile/')
 
-    @override_settings(
-        ACCOUNT_EMAIL_VERIFICATION=app_settings.EmailVerificationMethod
-        .MANDATORY,
-        ACCOUNT_AUTHENTICATION_METHOD=app_settings.AuthenticationMethod
-        .USERNAME)
-    def test_ajax_login_unverified(self):
-        user = get_user_model().objects.create(username='john', is_active=True)
-        user.set_password('doe')
-        user.save()
-        EmailAddress.objects.create(user=user,
-                                    email='john@example.com',
-                                    primary=True,
-                                    verified=False)
-        resp = self.client.post(reverse('account_login'),
-                                {'login': 'john',
-                                 'password': 'doe'},
-                                HTTP_X_REQUESTED_WITH='XMLHttpRequest')
```

Comment:
It's exactly the same test as `test_login_unverified_account_mandatory`. The fact that a view is requested with the `X-Requested-With: XMLHttpRequest` header shouldn't be tested for every single scenario IMO.